lib/airport.rb

```diff
@@ -1,2 +1,61 @@
+require 'plane'
+
 class Airport
+
+  attr_accessor :hanger
+
+  def initialize(hanger_capacity)
+    @hanger_capacity = hanger_capacity
+
+    @hanger = []
+    @sky  = []
+
+    while @hanger.length < hanger_capacity
+      @hanger << Plane.new
+    end
+
+    @hanger
+  end
+
+    def take_off
+      if (@hanger.any? && traffic_control?)
+        release_plane
+      end
+  end
+
+  def release_plane
+    released_plane = @hanger.pop
+    @sky << released_plane
+  end
+
+  def land
+    if (@hanger.length < @hanger_capacity && traffic_control?)
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.<br>Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.<br>Style/ParenthesesAroundCondition: Don't use parentheses around the condition of an `if`.